framework/src/main/java/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -1016,6 +990,26 @@ protected boolean visitIntersectionSupertype(
         return result;
     }
 
+    /**
+     * An intersection is a subtype if one of its bounds is a subtype of {@code supertype}.
```

Comment:
When this is called, have the primary annotations already been applied to every bound, so that each bound stands on its own without any need to refer to the primary annotations?

Reply:
Yes.  This method is only called on fully-annotated types.  That means that the intersection type has a primary annotation for every qualifier hierarchy.

Reply:
TOOD: Add comment.